src/Chapter4.hs

```diff
@@ -489,6 +507,26 @@ Implement the 'Applicative' instance for our 'List' type.
   type.
 -}
 
+instance Semigroup (List a) where
```

Comment:
Nice work on using standard typeclasses!

Reply:
Thanks! Yeah, I dabbled a little with Haskell in the past so I remember few things, among them are Semigroup and Monoid which are used for kind of aggregating things I believe. Thought could use it :)